test/clj/rems/api/test_audit_log.clj

```diff
@@ -29,8 +31,15 @@
             (testing "> authorized"
               (is (= 200 (:status (-> (request :get "/api/users/active")
                                       (authenticate "42" "owner")
+                                      handler)))))
+            (reset! time-a (time/now))
+            (testing "> application"
+              (is (= 200 (:status (-> (request :get (str "/api/applications/" app-id))
```

Comment:
Should we prefer `api-call` and `api-response` for these in the future? We also have the `(true? :success)` and `(= {:success true}` variants. Could add some idea task to unify these a bit.

Reply:
I think here it's more natural to use the explicit forms since some calls lack `authenticate`. I'll clean up a bit otherwise.